### Import Names 

In [52]:
import pandas as pd
# Read the CSV file into a pandas data frame (df)
heros = pd.read_csv('MarvelSuperHeros.csv', delimiter=',' ,header=None)

### Dataset Size

In [53]:
#number of super heros
heros.size

1491

### Lets have a look at few heros

In [54]:
heros.sample(10)

0
707                              Lord Tyger
828                              Molten Man
1042                       Robbie Robertson
1468                                X-Force
663                           Lady Bullseye
1189  Spider-Man (LEGO Marvel Super Heroes)
86                                Barracuda
347                                   Druig
157                                     Box
1400                                 Vision

### oops! Gotta get rid of those extra info in brackets

In [83]:
heros_extra=heros[0].str.split("(",expand=True)
heros_name_only=pd.DataFrame(heros_extra[0])

In [84]:
heros_name_only.sample(10)

0
211    Carol Danvers
1023       Red Hulk 
505        Hemingway
676   Lenny Balinger
1227       Stingray 
619         Joystick
1214         Starfox
147       Bloodstorm
914             Odin
403           Fixer

### Fixed! 

In [88]:
heros_name_only.to_csv("SuperHeroCorpus.csv",sep=',', encoding='utf-8', index=False, header=None)

### We now have a dataset of 1491 ~1.5K names.... Let's create our own Super Hero!